src/main/kotlin/Extension.kt

```diff
@@ -0,0 +1,8 @@
+fun String?.parseInt(): Int {
+    require(this != null) { "빈 값을 입력하면 안됩니다." }
+    try {
+        return this.toInt()
+    } catch (e: NumberFormatException) {
+        throw IllegalArgumentException("자연수로 변환하는데 실패했습니다.")
+    }
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
    require(!this.isNullOrBlank()) { "빈 값을 입력하면 안됩니다." }
    return this.toIntOrNull() ?: throw IllegalArgumentException("자연수로 변환하는데 실패했습니다.")
```

Reply:
이렇게 사용할수도 있군요! 감사합니다 👍 👍